dist.Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM openjdk:8u121
```

Comment:
Using this as the base image for the app fixes the issue with missing external files